man/addStat.cf.Rd

```diff
@@ -1,34 +0,0 @@
-\name{addStat.cf}
```

Comment:
why did you delete this?

Reply:
Sorry, this was a mistake, I now get this back, and augmented it with options reverse for the two arguments.